### DISCUSSION TITLE
Does backup & restore work for Docker (or K8 with docker engine) deployments?

### DISCUSSION THREAD

**Discussion by nlcsdev** [snipe/snipe-it#13252] [General]
I am testing the backup and restore feature from the GUI and there is a warning that states `(This does not currently work with S3 file storage or Docker.)`. Since I am running this on k8s and by extension on the docker engine, this warning was of concern to me.

I made an attempt to use the zip produced from a snipe-it application that was running on docker, and restored it on my snipe-it application that is on k8s. Everything looks ok, so I was wondering if that warning is outdated or is there something I should know?

**Comment by snipe** [comment]
Did you do that using the GUI or via command line?

**Reply by nlcsdev** [reply]
I did the backup & restore via the GUI.

**Reply by snipe** [reply]
Huh - it kinda shouldn't work, but if it does, awesome!

**Reply by nlcsdev** [reply]
What particular reasons make you believe it shouldn't work?

**Reply by snipe** [reply]
We tried it and it didn't lol

I don't remember changing much with Docker (or the filesystem layer within Snipe-IT) but I'm happy to change the language for that restore tool to make it less ominous. It can just be very challenging to troubleshoot these things without our own k8 setup.
